app/controllers/messages_controller.rb

```diff
@@ -6,7 +6,8 @@ def new
   def create
     raise Messages::ContentBlankError, nil if message_params[:content].blank?
 
-    @message = Message.new(message_params)
+    interface = Interface.create(interface_type: :web)
```

Comment:
This should `find` instead of `create`

```suggestion
    interface = Interface.create(interface_type: :web)
```

Reply:
We are now usgin Messages creator service!! Please take a look